api/ombpdf/management/commands/import_pdfs_from_policies.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from os.path import basename
+from typing import List, Optional, Set, Tuple
+
+import requests
+from django.core.management.base import BaseCommand
+from django.db.models import Exists, OuterRef
+from requests.exceptions import RequestException
+
+from document.models import DocNode
+from ombpdf.management.commands.scrape_memoranda import Url, parse_pdf
+from reqs.models import Policy
+
+OLD_DOMAIN = 'www.whitehouse.gov'
+NEW_DOMAIN = 'obamawhitehouse.archives.gov'
+PROBLEM_PDFS = (
+    'a11_2016.pdf',
+)
+logger = logging.getLogger(__name__)
+
+
+def best_url(urls: List[Url]) -> Optional[Url]:
+    """Try hitting the requested url; if it fails, fall back to an
+    alternate."""
+    for url in urls:
+        if basename(url) in PROBLEM_PDFS:
+            continue
+        try:
+            response = requests.head(url)
+            if response.status_code == 200:
+                return url
+        except RequestException:
+            continue
+    return None
+
+
+def import_from_policy(policy: Policy) -> bool:
+    """Convert a policy to a list of potential urls to try, pick the first
+    that works, then attempt to parse."""
+    uri = policy.uri.strip()
+    urls_to_try = [Url(uri)]
+    if OLD_DOMAIN in uri:
+        urls_to_try.append(Url(uri.replace(OLD_DOMAIN, NEW_DOMAIN)))
+    if policy.document_source:
+        urls_to_try.insert(0, Url(policy.document_source.url))
+
+    url = best_url(urls_to_try)
+    return bool(url and parse_pdf(policy, url))
```

Comment:
I should have mentioned this when reviewing #801, but you're welcome to use [`tqdm`](https://pypi.python.org/pypi/tqdm) in `parse_pdf` to show download progress for the PDFs. I added it to our dev requirements file when migrating ombpdf to this repo, since it's used by that package's PDF retriever thingy.

Reply:
Cool! I'll give it a quick go; if it's simple, I'll update this PR.

Reply:
I'll add in an addendum PR as it'll require moving tqdm to the prod requirements.